src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -132,25 +106,80 @@ private bool ValidateOptions(FileWorkItemsOptions options, IFileSystem fileSyste
             // We convert it to a URI with the ctor Uri(string, UriKind.RelativeOrAbsolute).
             // If it succeeds, we can assign the result to a Uri-valued property; if it fails,
```

Comment:
>If it succeeds, we can assign the result to a Uri-valued property [](start = 15, length = 65)

We no longer do that; you removed the `Uri`-valued property from the options class.

Reply:
We still do this, we've just moved the URI-valued property from the options object to the context object. I clarified this in the comment.

---
In reply to: [381016469](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381016469) [](ancestors = 381016469)